### PR TITLE
feat(others) add monkey test.

### PR DESCRIPTION
**Pull request by FASTSHIFT**
### Description of the feature or fix

A simple monkey test to stress test the application.
https://github.com/lvgl/lvgl/issues/2276#issuecomment-915040816

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [x] Update the documentation

### CODE REVIEW

File: src/extra/others/monkey/lv_monkey.c [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    data->state = monkey->indev_data.state;
+}
+
+static int32_t lv_monkey_random(int32_t howsmall, int32_t howbig)
+{
+    if(howsmall >= howbig) {
+        return howsmall;
+    }
+    int32_t diff = howbig - howsmall + 1;
+    return rand() % diff + howsmall;
```
Comment (kisvegabor) [inline src/extra/others/monkey/lv_monkey.c]: Can we use `lv_rand` here?

File: src/extra/others/monkey/lv_monkey.c [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    switch(monkey->indev_drv.type) {
+        case LV_INDEV_TYPE_POINTER:
+            data->point.x = (lv_coord_t)lv_monkey_random(0, LV_HOR_RES - 1);
+            data->point.y = (lv_coord_t)lv_monkey_random(0, LV_VER_RES - 1);
+            break;
+        case LV_INDEV_TYPE_ENCODER:
+            data->enc_diff = (int16_t)lv_monkey_random(monkey->config.input_range.min, monkey->config.input_range.max);
+            break;
+        case LV_INDEV_TYPE_BUTTON:
+            data->btn_id = (uint32_t)lv_monkey_random(monkey->config.input_range.min, monkey->config.input_range.max);
```
Comment (kisvegabor) [inline src/extra/others/monkey/lv_monkey.c:171]: If `lv_indev_set_button_points()` is not used to set an array that maps IDs to x/y coordinates of the screen, it won't do anything. I think the docs should mention that this step needs to be done manually using `lv_monkey_get_indev`.

**Review by kisvegabor — COMMENTED** [review]
Looks good, thank you!

I had only two minor comments.

File: src/extra/others/monkey/lv_monkey.c (hunk 2 above)
Comment (FASTSHIFT) [reply src/extra/others/monkey/lv_monkey.c:171]: Done

File: src/extra/others/monkey/lv_monkey.c (hunk 1 above)
Comment (FASTSHIFT) [reply src/extra/others/monkey/lv_monkey.c]: Done

**Comment by FASTSHIFT** [comment]
@kisvegabor 
ready to merge?

**Comment by kisvegabor** [comment]
Yes, all looks good. Thank you!
I had some busy days. Sorry that I was sluggish with answering.

**Pull request merged**